### PR TITLE
Defining the main function

### PR DESCRIPTION
**Pull request by mamkhari**
This PR does the following:
- Adds the few  variable values, one of them is __name__ whose value is set as __main__.
-  And then it executes all of the code found in the file.
 
Motivation: if __name__ == "__main__" is useful when  you want to execute some code only if the file was run directly, and not imported.

### CODE REVIEW

**Comment by mamkhari** [comment]
This PR does the following:

    Adds the few variable values, one of them is name whose value is set as main.
    And then it executes all of the code found in the file.

Motivation: if name == "main" is useful when you want to execute some code only if the file was run directly, and not imported.

**Comment by mamkhari** [comment]
How do I PR title?

File: extract_unique_sensors_01.py [hunk 1]
```diff
@@ -87,3 +87,9 @@ def splitting_extracted_sensors(candidate_sensors, sensor_name):
         sensors_list.append(sensor_)
     return sensors_list
 
+if __name__ == "__main__":
+    sensors_data = read_sensors(file_name, url)
+    extracted_sensors = extract_all_sensors(sensors_data, kat_sensor)
+    unique_sensors_names = splitting_extracted_sensors(extracted_sensors, kat_sensor)
+    
+
```
Comment (mmphego) [inline extract_unique_sensors_01.py]: Too many empty lines.
Aren't you supposed to print out the uniq sensors?

**Comment by mmphego** [comment]
> How do I PR title?

Click on the ellipses (...)
image

File: extract_unique_sensors_01.py (hunk 1 above)
Comment (mamkhari) [reply extract_unique_sensors_01.py]: Oh yes, I'm supposed to call the print() in the if __name__ condition. I will add that, and remove empty lines.

File: extract_unique_sensors_01.py [hunk 2]
```diff
@@ -91,5 +91,5 @@ def splitting_extracted_sensors(candidate_sensors, sensor_name):
     sensors_data = read_sensors(file_name, url)
     extracted_sensors = extract_all_sensors(sensors_data, kat_sensor)
     unique_sensors_names = splitting_extracted_sensors(extracted_sensors, kat_sensor)
-    
+print(unique_sensors_names)
```
Comment (mmphego) [inline extract_unique_sensors_01.py]: indent!

File: extract_unique_sensors_01.py [hunk 3]
```diff
@@ -87,3 +87,9 @@ def splitting_extracted_sensors(candidate_sensors, sensor_name):
         sensors_list.append(sensor_)
     return sensors_list
 
+if __name__ == "__main__":
+    sensors_data = read_sensors(file_name, url)
+    extracted_sensors = extract_all_sensors(sensors_data, kat_sensor)
+    unique_sensors_names = splitting_extracted_sensors(extracted_sensors, kat_sensor)
+    print(unique_sensors_names)
+
```
Comment (mmphego) [inline extract_unique_sensors_01.py]: Delete empty line!

**Review by mmphego — APPROVED** [review]

**Pull request merged**